SimpleSDMPolygons/src/SimpleSDMPolygons.jl

```diff
@@ -0,0 +1,59 @@
+module SimpleSDMPolygons
+
+const _data_storage_folders = first([
+    Base.DEPOT_PATH...,
+    homedir(),
+])
+
+const _POLYGON_PATH = get(ENV, "SDMLAYERS_PATH", joinpath(_data_storage_folders, "SimpleSDMPolygons"))
+isdir(_POLYGON_PATH) || mkpath(_POLYGON_PATH)
+
+using Downloads
+using ZipArchives
+using Proj 
+using GeoJSON
+using HTTP
+using DataFrames, CSV 
+
+import GeoJSON as GJ 
```

Comment:
You can do `const GJ = GeoJSON`  and achieve the same result with no performance cost as `GeoJSON` is aleady in use

Reply:
done